packages/compass/src/main/application.ts

```diff
@@ -160,12 +159,6 @@ class CompassApplication {
     });
   }
 
-  private static setupJavaScriptArguments(): void {
-    // For Linux users with drivers that are avoided by Chromium we disable the
-    // GPU check to attempt to bypass the disabled WebGL settings.
-    app.commandLine.appendSwitch('ignore-gpu-blacklist', 'true');
```

Comment:
Feels like it's something we want to set as early as possible, for consistency moved this and the new `appendSwitch` call to the setup-* file that is the first thing that runs in the app